paparazzi-gradle-plugin/build.gradle

```diff
@@ -45,6 +45,12 @@ buildConfig {
 }
 
 tasks.withType(Test).configureEach {
+  // These are referenced with relative paths in src/test/projects/test.settings.gradle.
+  // They need to be declared on the Test task to make sure it runs with the new files.
+  inputs.file(rootProject.layout.projectDirectory.file("gradle/libs.versions.toml"))
+    .withPropertyName("libs.versions.toml")
+  inputs.dir(rootProject.layout.buildDirectory.dir("localMaven"))
```

Comment:
Drive-by comments, this will never be up to date because publication tasks include timestamps that get generated every time maven-metadata.xml

Reply:
@liutikas See the other changed build.gradle file, the publishing task will be up to date, because the input to it didn't change. Publishing task doesn't declare outputs, nor "current time" as input. So it won't rewrite the maven-metadata.xml with the timestamp on every execution when we just simply declare the folder as the output.

Reply:
Hmm, just realized that the other script is declaring overlapping outputs into one folder, wonder why it didn't fail.